airbyte-ci/connectors/pipelines/pipelines/dagger/containers/internal_tools.py

```diff
@@ -37,3 +37,17 @@ async def with_connector_ops(context: PipelineContext) -> Container:
     python_base_environment: Container = with_python_base(context)
 
     return await with_installed_pipx_package(context, python_base_environment, INTERNAL_TOOL_PATHS.CONNECTOR_OPS.value)
+
+
+async def with_connectors_qa(context: PipelineContext) -> Container:
```

Comment:
For my personal knowledge: how is this used?

Reply:
It's not used, I should remove it. It's not necessary as it's the `SimpleDockerStep` which will take care of installing the `connectors_qa` package.